uktrade/data-hub-frontend
# Implement redux on the form to add interactions

## Description of change

Add redux store to the new interaction form.

## Test instructions

1. Go to https://data-hub-fro-interactio-r1yy7f.herokuapp.com/interactions/create?company_id=5bb239e4-b227-4a84-bc2e-3420fc129447
2. Confirm if the form works the same as https://data-hub-fro-interactio-r1yy7f.herokuapp.com/companies/5bb239e4-b227-4a84-bc2e-3420fc129447/interactions/create
